jaxrs-spec/src/main/asciidoc/chapters/appendix/_bibliography.adoc

```diff
@@ -2,33 +2,33 @@
 == Bibliography
 
 - [[[bib1,1]]] R. Fielding. Architectural Styles and the Design of Network-based Software Architectures. Ph.d
-               dissertation, University of California, Irvine, 2000. See http://roy.gbiv.com/pubs/dissertation/top.htm.
+               dissertation, University of California, Irvine, 2000. See https://roy.gbiv.com/pubs/dissertation/top.htm.
 
-- [[[bib2,2]]] REST Wiki. Web site. See http://rest.blueoxen.net/cgi-bin/wiki.pl.
+- [[[bib2,2]]] REST Wiki. Web site. See https://rest.blueoxen.net/cgi-bin/wiki.pl.
```

Comment:
This link is broken. The host is unknown, it cannot be reached, no matter http or https. What to do then? @mkarg

Reply:
Keep it as HTTP. In a seprate discussion we need to find replacement.

Reply:
Done